Source/loadsave.cpp

```diff
@@ -207,42 +207,537 @@ BOOL OLoad()
 		return FALSE;
 }
 
+void CopyInt(const void *src, void *dst)

```

Comment:
Could we really do `memcpy` here if we're going to support big-endian systems? (#70)
Shouldn't it be implement similar to original `WLoad` then?
If that's the case it probably would not work with `void*`->`void*` interface then unfortunately

Reply:
I would also suggest to use `template`s here, that would make implementation more concise and allow to not hardcode sizes, but this may be done as an improvement later of course.

Reply:
A endian safe implementation would definatly be prefered, but maybe not required initially since the whole engine needs endian work.